src/fungible/amount.rs

```diff
@@ -25,7 +25,8 @@
 //! floating point amount with `*_amount` suffix, and reserve `*_value` suffix
 //! for methods returning atomic value.
 
-use std::ops::{Add, AddAssign};
+use std::convert::TryFrom;
```

Comment:
We are already on edition 2021, where importing `TryFrom` is not required anymore